tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -35,6 +37,23 @@ def run_optimize(args: Tuple[str, str]) -> None:
     study.optimize(objective, n_trials=20)
 
 
+def run_optimize_with_progbar(args: Tuple[str, str, int]) -> str:
+    study_name = args[0]
+    storage_url = args[1]
+    n_jobs = args[2]
+    # Create a study
```

Comment:
```suggestion

    # Create a study
```

Reply:
I don't think that blank line is necessary because existing ```run_optimize``` does not have blank line before comment ```# Create a study```.